cg/services/deliver_files/factory.py

```diff
@@ -142,11 +144,13 @@ def _get_file_tag_fetcher(
         """
         if delivery_destination == DeliveryDestination.FOHM:
             return FOHMUploadTagsFetcher()
+        # Do we need different tag fetchers now that BAM is included in the raw data tags?
         service_map: dict[DataDelivery, Type[FetchDeliveryFileTagsService]] = {
             DataDelivery.FASTQ: SampleAndCaseDeliveryTagsFetcher,
             DataDelivery.ANALYSIS_FILES: SampleAndCaseDeliveryTagsFetcher,
             DataDelivery.FASTQ_ANALYSIS: SampleAndCaseDeliveryTagsFetcher,
             DataDelivery.BAM: BamDeliveryTagsFetcher,
+            DataDelivery.RAW_DATA_ANALYSIS: SampleAndCaseDeliveryTagsFetcher,
```

Comment:
We might not need different tag fetchers now as BAM and FASTQ are in the same set of tags. Still need to look deeper on it

Reply:
fixed